src/Backend/Modules/Pages/Domain/Page/Command/CopyPagesToOtherLocaleHandler.php

```diff
@@ -23,12 +23,18 @@ public function handle(CopyModuleToOtherLocaleCommandInterface $command): void
         $fromLanguage = (string) $command->getFromLocale();
 
         // Get already copied ContentBlock ids
-        $contentBlockIds = $command->getPreviousResults()->getExtraIds('ContentBlocks');
-        $contentBlockOldIds = array_keys($contentBlockIds);
+        $hasContentBlocks = $command->getPreviousResults()->hasModule('ContentBlocks');
+        if ($hasContentBlocks) {
+            $contentBlockIds = $command->getPreviousResults()->getExtraIds('ContentBlocks');
+            $contentBlockOldIds = array_keys($contentBlockIds);
+        }
 
         // Get already copied Location ids
-        $locationWidgetIds = $command->getPreviousResults()->getExtraIds('Location');
-        $locationWidgetOldIds = array_keys($locationWidgetIds);
+        $hasLocations = $command->getPreviousResults()->hasModule('Location');
```

Comment:
you should create a `$previousResults` variable since you use `$command->getPreviousResults()` plenty of times

Reply:
Fixed